crates/node/src/interactive_cli/webui.rs

```diff
@@ -0,0 +1,31 @@
+use calimero_network::config::DEFAULT_PORT;
+use clap::Args;
+use eyre::Result;
+use libp2p::multiaddr::Protocol;
+use webbrowser;
+
+use crate::Node;
+
+#[derive(Debug, Args)]
+pub struct WebUICommand;
+
+impl WebUICommand {
+    pub fn run(&self, node: &Node) -> Result<()> {
+        let port = node
+            .server_config
+            .listen
+            .iter()
+            .find_map(|addr| {
+                addr.iter().find_map(|proto| match proto {
+                    Protocol::Tcp(port) => Some(port),
+                    _ => None,
+                })
+            })
+            .unwrap_or(DEFAULT_PORT);
+
+        let url = format!("http://localhost:{}/admin-dashboard", port);
```

Comment:
Consider reusing this from meroctl:

https://github.com/calimero-network/core/blob/50dbf0736a55ed0c48328ef524d3e472707b8677/crates/meroctl/src/common.rs#L26-L48

We can move it to [`crates/primitives/src/common.rs`](https://github.com/calimero-network/core/blob/50dbf0736a55ed0c48328ef524d3e472707b8677/crates/primitives/src/common.rs)